chain/client/src/client_actor.rs

```diff
@@ -1767,6 +1775,37 @@ impl ClientActor {
         }
     }
 
+    /// Verifies if the node possesses sync block.
+    /// It is the last block of the previous epoch.
+    /// If the block is absent, the node requests it from peers.
+    fn sync_block_status(
+        &self,
+        prev_hash: &CryptoHash,
+        now: DateTime<Utc>,
+    ) -> Result<(bool, bool), near_chain::Error> {
+        let (request_block, have_block) = if !self.client.chain.block_exists(prev_hash)? {
+            let timeout = self.client.config.state_sync_timeout.as_secs() as i64;
```

Comment:
what's the reason for rounding down to the nearest whole seconds? If I change my config to say 3.8 seconds, why wouldn't we want to respect it here?